src/pretix/control/views/organizer.py

```diff
@@ -195,13 +198,50 @@ class OrganizerUpdate(OrganizerPermissionRequiredMixin, UpdateView):
     model = Organizer
     form_class = OrganizerUpdateForm
     template_name = 'pretixcontrol/organizers/edit.html'
-    permission = None
+    permission = 'can_change_settings'
```

Comment:
This is useless, as ``OrganizerPermission`` currently has no ``can_change_settings`` attribute. (The permission system is not that well-polished, we know...)